features/change_files_spec.rb

```diff
@@ -6,6 +6,8 @@
   subject(:forecast) do
     Crystalball.foresee(workdir: root, map_path: root.join('execution_map.yml')) do |predictor|
       predictor.use Crystalball::Predictor::ModifiedExecutionPaths.new
+      predictor.use Crystalball::Predictor::DirectMapPredictor.new from: %r{lib/models/(?<file>.*).rb},
+                                                                   to: './spec/models/%<file>s_spec.rb'
```

Comment:
maybe it worth to have separate feature file per each prediction strategy

Reply:
makes sense